types/ramda/index.d.ts

```diff
@@ -1196,19 +1197,20 @@ declare namespace R {
         pair<F, S>(fst: F, snd: S): [F, S];
 
         /**
-         * Accepts as its arguments a function and any number of values and returns a function that,
-         * when invoked, calls the original function with all of the values prepended to the
-         * original function's arguments list. In some libraries this function is named `applyLeft`.
+         * Takes a function `f` and a list of arguments, and returns a function `g`.
+         * When applied, `g` returns the result of applying `f` to the arguments
+         * provided initially followed by the arguments provided to `g`.
          */
-        partial(fn: (...a: any[]) => any, ...args: any[]): (...a: any[]) => any;
-        partial<T>(fn: (...a: any[]) => T, ...args: any[]): (...a: any[]) => T;
+        partial(fn: (...a: any[]) => any, args: any[]): (...a: any[]) => any;
```

Comment:
This should be removed because the end result should have the same type as `fn`'s return type instead of `any`